app/assets/stylesheets/ama_layout/layout/helper-classes.scss

```diff
@@ -533,6 +533,70 @@ button[disabled],
   word-break: break-word;
 }
 
+.semi-bold{
+  font-weight: 600;
+
+  &--small{
+    @extend .semi-bold;
+    font-size: $base-font-size*0.75;
```

Comment:
I think you can set all of these to the variables you created: $small-font-size, etc.